docs/_static/vdr/v2.yaml

```diff
@@ -298,7 +298,8 @@ paths:
       summary: Delete a specific service from the subject
       description: |
         Removes the service from all DID Documents in the subject. Matching is done on the fragment of the id.
-        No cascading will happen for references to the service. 
+        No cascading will happen for references to the service.
+        Make sure to only URL encode the pound (#) as %23 in the serviceId. Do not encode the colons (:). 
```

Comment:
isn't this default behavior? Standard API clients should do this?

Reply:
If I do an encode URI component in Postman, it escapes both `:` and `#` so not all tools are equal